src/mempolicy.c

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+ * Copyright (C) 2023-2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+#include <umf/mempolicy.h>
+
+#include "base_alloc_global.h"
+#include "mempolicy.h"
```

Comment:
you include "mempolicy.h" two times - please leave only <umf/mempolicy.h> version

Reply:
Done